uci.go

```diff
@@ -34,6 +34,10 @@ type Tree interface {
 	// internal memory and does not access the file system.
 	Revert(configs ...string)
 
+	// GetSections returns the names of all sections of a certain type
+	// in a config, and a boolean indicating wheter the config file exists.
```

Comment:
Small typo:

```diff
- wheter
+ whether
```

Reply:
Thanks. I'm really bad at spotting typos in my comments.